src/plugins/lowercase-directories.ts

```diff
@@ -0,0 +1,23 @@
+import { rename } from "fs";
```

Comment:
```suggestion
import { rename } from "fs/promises";
```

We can use the promise-based package.